ingest/indexer/service/blockprocessor/export_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 
 	commondomain "github.com/osmosis-labs/osmosis/v26/ingest/common/domain"
+	commonservice "github.com/osmosis-labs/osmosis/v26/ingest/common/service"
```

Comment:
Would it make make codebase more easier to undersand/follow by moving contents of `common` to root of `ingest`? 
For example:

 `github.com/osmosis-labs/osmosis/v26/ingest/common/domain` -> `github.com/osmosis-labs/osmosis/v26/ingest/domain`.

As a side effect we would endup with more meaningful import paths/package names: 

```
	commonservice "github.com/osmosis-labs/osmosis/v26/ingest/common/service"
```

vs

```
	ingestservice "github.com/osmosis-labs/osmosis/v26/ingest/service"
```

IMHO "common" packages tend to acumulate all sorts of code with different responsibilites instead creating focused packages, and this package does not align with general `osmosis` codebase, there are no other "common" packages in the codebase.

Reply:
Hey Deividas, under the `ingest` folder, we have `common`, `sqs` and `indexer` folders, where `common` are the shared packages for the ingester `sqs` and `indexer`. Therefore, I don't think we should move the `common` from its current location to `indexer` as `common` is needed by `sqs` as well. For the `export_test.go`, to make it easier to read, I will define an alias for the indexer domain so that readability is improved.
I hope I address the issue you raise. Thanks!

Reply:
Hey Calvin! Thanks for sharing your thoughts. Just to make sure you got it right, the idea I had was to move to move contents of `common` package to the root of `ingest`. As a result we would end up, for example, frorm current:

```
ingest
├── common
│   ├── domain
│   └── service
├── indexer
└── sqs
```

To:

```
ingest
├── domain
├── indexer
├── service
└── sqs
```

Feel free to mark this thread as resolved if you think we still should not do that. :pray: